### DISCUSSION TITLE
2022-01-05 Weekly Meeting 37 Agenda

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#156] [Weekly Meetings]
Weekly Meeting 37 is happening on Wednesday January 5th, 2022 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Agenda
• Volunteer to take notes? Volunteer to record?
• Welcome newcomers
• Review Working Group Meetings
• GLTF Subgroup Updates
• Reminder the next chair election is Feburary 2nd
  • Created [this new discussion](https://github.com/omigroup/omigroup/discussions/157) to talk openly about what the future chair structure could look like. 
• Schedule our next retro (next week or the following)
• Open Floor (final 15min)

Working groups and Upcoming Events: :calendar: [Add the OMI event calendar to your google calendar](https://calendar.google.com/calendar/u/4?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t)
- Website Working Group - Tuesday Jan 4, 2021 at 3pm - 5pm PT / 6pm - 8pm ET (contacts: Shadow and Echoparadigm)
- OMI Weekly Meeting 37 - Wednesday Jan 5, 2021 at 9am - 10am PT / 12pm - 1pm ET (contacts: antpb for agenda, mrmetaverse for questions)
- gLTF Extensions Working Group - Thursday Jan 6th at 2:30 - 3:30pm PT / 5:30 - 6:30pm ET (contact Fire or Robert)
- OMI-Chan Avatar Working Group - Saturday January 8th at 12:30 - 1:30 pm PT / 3:30 - 4:30 pm ET (contact Nova)
- WASM Working group - TBA (contact Adam Frisbee[lead], or humbletim or Lyuma or Robert) 
- Interoperator working group? 

- any others? (Metaverse traversal? Metamapping? etc.)

**Comment by mrmetaverse** [comment]
JustI got this weeks agenda _started_ however antpb is set to create next weeks agenda. Again I recommend visiting previous formats from a few weeks ago. 

feel free to delete this image from the main description :point_down: 
![image](https://user-images.githubusercontent.com/63426722/147790741-c058ffe1-8bd2-4cfe-b221-ef4182a8e26e.png)

Just wanted to highlight that we have a calendar, and we have several events slated to occur next week. Heads up @technobaboo the omi avatar group is scheduled to meet on the 8th, so I encourage you to use the next week drumming up support / attendance for that.

**Reply by mrmetaverse** [reply]
Remove image because it's Tuesday not Monday for website group 😸

**Reply by mrmetaverse** [reply]
WASN will meet at the earliest around the 12th but the date is still TBD

**Reply by mrmetaverse** [reply]
@technobaboo following up with you about the omi-chan group

**Comment by mrmetaverse** [comment]
Reminder for newcomers: working group times are established by the participants. We have contributors all around the world. If you can't make a time work, you suggest a new time to the working group. No centralized leadership sets times and we are aware that we are not solely consistent of North American contributors.

**Reply by mrmetaverse** [reply]
But if you decide on a new time, you need to attend. Setting working group meeting times is difficult enough to accommodate everyone. If we have enough participation it's possible there may be multiple meetings for multiple working groups but that is a good problem to have and one that can be addressed when there are enough participants.

**Comment by antpb** [comment]
@mrmetaverse I was thinking we could start each meeting with a script to consistently orient the goal of the meeting for everyone attending (new and existing members.) 

> Intro

> “The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. Aside from technical work, OMI aims to create a community of artists, creators, developers, and other innovators to discuss and explore concepts surrounding the design and development of virtual worlds.
> If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!
> 
> We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.
> 
> Before we get started would anyone like to volunteer to take notes?”
> 

If someone volunteers they can take notes in the comment section { 

> “Please feel free to ask if we should note something if it is not clear what should be logged. The note taker is granted the ability to stop any conversation to back up and agree on consensus or action items.”


If no volunteer { Meeting facilitator takes notes.

**Reply by antpb** [reply]
oh, I think I need permissions to edit this discussion description.

**Reply by mrmetaverse** [reply]
oh ok I will add you to the team

**Reply by mrmetaverse** [reply]
this is great btw!

**Comment by mrmetaverse** [comment]
++ Do we want to schedule a retro?

**Comment by robertlong** [comment]
Also worth noting, the next chair election is Feburary 2nd (6 months from our last one on August 4th).

**Comment by mrmetaverse** [comment]
https://github.com/omigroup/omigroup/issues/109

**Comment by mrmetaverse** [comment]
This _might_ be too lengthy for tomorrow and may require another session or a heads up that we are going to talk about this, but [I created a discussion to think about our future structure](https://github.com/omigroup/omigroup/discussions/157)

**Comment by mrmetaverse** [comment]
Meeting started, we welcomed a lot of newcomers! 
Reviewed working groups model 
Checked in with the omigroup/website group 
Checked in with the gltf working group 
Upcoming meetings for omi-chan and wasm working groups 
Q: What is wasm? https://webassembly.org/

**Comment by mrmetaverse** [comment]
created this meta issue to update our readme to include links to the repos of each active working group. Clean readmes on each working group would be useful for newcomers who want to get involved. https://github.com/omigroup/omigroup/issues/158

**Comment by robertlong** [comment]
Make to move to UTC time for meetings / documentation.

**Comment by mrmetaverse** [comment]
New WASM Group meets today: 

https://github.com/omigroup/omi-wasm-group/discussions/8

New PR Ready for review: 

https://github.com/omigroup/omigroup/pull/159
